ambari-server/src/main/java/org/apache/ambari/server/state/cluster/ClusterImpl.java

```diff
@@ -1018,13 +1018,21 @@ public void addServiceGroup(ServiceGroup serviceGroup) {
   }
 
   @Override
-  public ServiceGroup addServiceGroup(String serviceGroupName) throws AmbariException {
+  public ServiceGroup addServiceGroup(String serviceGroupName, String version) throws AmbariException {
     if (serviceGroups.containsKey(serviceGroupName)) {
```

Comment:
addServiceGroup(serviceGroupName, stackId) already has the same check. Remove duplicate code.

Reply:
I will remove the check in addServiceGroup(String serviceGroupName, String version)